godot-macros/src/godot_api.rs

```diff
@@ -210,7 +214,7 @@ fn process_godot_fns(decl: &mut Impl) -> Result<(Vec<Function>, Vec<Function>),
             continue;
         };
 
-        if let Some(attr) = extract_attributes(&method, &method.attributes)? {
+        if let Some(attr) = extract_attributes(&method, &method.attributes.clone())? {
```

Comment:
This change is necessary because otherwise, the compiler complains about 2 immutable borrows happening here and below via `method.attributes.remove(attr.index);`

Reply:
This is because the return type of `extract_attributes`, `BoundAttr`, now contains a lifetime. Remove that as mentioned above and code becomes easier 🙂